lerna.json

```diff
@@ -2,11 +2,6 @@
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "version": "3.2.4",
   "npmClient": "yarn",
-  "packages": [
-    "packages/*",
```

Comment:
is this the default? Do we not want to keep `packages/*` around for the standard bb packages?